### ISSUE TITLE
Any plans to update this to Babel 6?

### ISSUE THREAD

**Issue by aluxian** [shuhei/babel-plugin-auto-assign#2]
How to update: see `Plugin API` on http://babeljs.io/blog/2015/10/29/6.0.0/

**Comment by shuhei** [comment]
Yes, I'll support Babel 6 by the end of 2015. I haven't updated it because [decorators were removed in Babel 6 because the decorator proposal has vastly changed from the version that's in Babel 5](https://phabricator.babeljs.io/T2645#65806).

Now we have [babel-plugin-transform-decorators-legacy](https://www.npmjs.com/package/babel-plugin-transform-decorators-legacy) that supports Babel 5 compatible decorators. My plan is to support decorators with the plugin until Babel 6 supports the new spec of decorators.

**Comment by aluxian** [comment]
Great! Thank you :)

**Comment by McGiogen** [comment]
Any news about this?

**Comment by jakewhelan** [comment]
would like to use this plugin with babel 7!

**Comment by shuhei** [comment]
Sorry, I don't use this plugin by myself, and won't maintain it anymore. If you really need it, it shouldn't be so hard to create one from scratch with babel 6/7 API.
